packages/story-editor/src/app/rightClickMenu/constants.js

```diff
@@ -59,6 +59,7 @@ export const RIGHT_CLICK_MENU_LABELS = {
   DUPLICATE_PAGE: __('Duplicate Page', 'web-stories'),
   DUPLICATE_ELEMENTS: (numElements = 1) =>
     _n('Duplicate Element', 'Duplicate Elements', numElements, 'web-stories'),
+  LOCK_UNLOCK: __('Lock/unlock', 'web-stories'),
```

Comment:
Small typo, but aside from that I was wondering if we should use separate labels - I've tested RCM with VoiceOver. @agingoldseco @aaskedall 

![image](https://user-images.githubusercontent.com/21036224/165515483-a081882f-e064-4d1c-97c8-77f353a931af.png)

[Figma](https://www.figma.com/file/dc9xrVlRhQq4KTE9WQzU1R/_Editor---Sprint-Feature-Level-Designs_?node-id=9935%3A421370)

Reply:
The design in Figma just had Lock/Unlock as a single label. And Figma itself uses only a single label as well, so I feel that is fine. We can always change it later. I fixed the capital letter in 85da1ebab8.

Reply:
Alright, just note that Figma has zero a11y (VoiceOver only detects the main window) so it's not a benchmark for us.